src/domain/shared/components/search-cards/base/SearchBaseCard.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { ComponentPropsWithRef, FC, useMemo } from 'react';
+import { CardMediaProps, styled, SvgIconProps } from '@mui/material';
+import Card from '@mui/material/Card';
+import CardMedia from '@mui/material/CardMedia';
+import CardContent from '@mui/material/CardContent';
+import CardActions from '@mui/material/CardActions';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import TagsComponent from '../../TagsComponent/TagsComponent';
+import { ClampedTypography } from '../../ClampedTypography';
+import { RouterLink } from '../../../../../common/components/core/RouterLink';
+import Link from '@mui/material/Link';
+
+const MATCH_TERMS_HEIGHT = 50;
```

Comment:
maybe these could go in the theme constants file too

Reply:
I really don't know. These are constants are related to the base component itself. Which will not be used directly.